src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/healthcert/PersonName.kt

```diff
@@ -20,4 +20,14 @@ data class PersonName(
 	@Json(name = "fnt") val standardizedFamilyName: String,
 	@Json(name = "gn") val givenName: String?,
 	@Json(name = "gnt") val standardizedGivenName: String?,
-) : Serializable
\ No newline at end of file
+) : Serializable {
+
+	fun prettyFamilyName(): String = familyName ?: standardizedFamilyName
+
+	fun prettyGivenName(): String = givenName ?: (standardizedGivenName ?: "")
```

Comment:
I'm wondering if the SDK should be the one to decide what string to fall back if both `givenName` and `standardizedGivenName` are null. Personally I'd prefer the SDK to return null in those cases and let the application decide on a default string. Otherwise the app ends up requiring a blank string check or wrongly thinking the name is present when it isn't.

Reply:
I think the prefix `pretty` has this covered. If you just want to print it, you can use this opinionated method, at the cost of this method's opinion what "pretty" means.
If you want more control, you can always access the fields yourself.